### PR TITLE
Increase judgment for debug log

### PR DESCRIPTION
**Pull request by totalo**

### CODE REVIEW

**Comment by menghaoranss** [comment]
@totalo There are conflicting files.

**Comment by totalo** [comment]
> @totalo There are conflicting files.

fixed! thx~

File: shardingsphere-agent/shardingsphere-agent-core/src/main/java/org/apache/shardingsphere/agent/core/bytebuddy/listener/LoggingListener.java [hunk 1]
```diff
@@ -35,7 +35,9 @@ public void onDiscovery(final String typeName, final ClassLoader classLoader, fi
     
     @Override
     public void onTransformation(final TypeDescription typeDescription, final ClassLoader classLoader, final JavaModule module, final boolean loaded, final DynamicType dynamicType) {
-        log.debug("On transformation class {}", typeDescription.getTypeName());
+        if (log.isDebugEnabled()) {
+            log.debug("On transformation class {}", typeDescription.getTypeName());
```
Comment (terrymanu) [inline shardingsphere-agent/shardingsphere-agent-core/src/main/java/org/apache/shardingsphere/agent/core/bytebuddy/listener/LoggingListener.java:39]: Is it necessary? the symbol `{}` has already avoid assembly the strings.

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: shardingsphere-agent/shardingsphere-agent-core/src/main/java/org/apache/shardingsphere/agent/core/bytebuddy/listener/LoggingListener.java (hunk 1 above)
Comment (totalo) [reply shardingsphere-agent/shardingsphere-agent-core/src/main/java/org/apache/shardingsphere/agent/core/bytebuddy/listener/LoggingListener.java:39]: I think that if its log level is higher than debug, it is unnecessary for the parameter in the debug method to be a method call.

**Pull request closed without merging**